packages/react-native-analytics/src/integrations/AnalyticsService/AnalyticsService.js

```diff
@@ -12,17 +12,19 @@
  * @subcategory Integrations
  */
 
-import { integrations, trackTypes } from '@farfetch/blackout-core/analytics';
+import { trackTypes } from '@farfetch/blackout-core/analytics';
+import CoreAnalyticsService from '@farfetch/blackout-core/analytics/integrations/AnalyticsService/AnalyticsService'; // TODO: The folder of the integration does not contain an index.js file. Remove this when the index.js file is added.
+
```

Comment:
Just a tip, add the comment above the line, it is more readable and doesn't split columns when IDE's/Editors are set to 80/100.

Reply:
updated. thanks 🙂